integration-tests/suites/listening_ports.go

```diff
@@ -130,3 +142,15 @@ func (s *ProcessListeningOnPortTestSuite) TestProcessListeningOnPort() {
 func getProcessListeningOnPortsImage() string {
 	return config.Images().QaImageByKey("qa-plop")
 }
+
+func (s *ProcessListeningOnPortTestSuite) openPort(port uint16) {
+	res, err := http.Get(fmt.Sprintf("http://%s:%s/open/%d", s.serverIp, s.serverPort, port))
```

Comment:
[nit] can we store the url in the suite?

e.g.

```
res, err := http.Get(fmt.Sprintf("%s/open/%d", s.serverURL, port))
```

now that I've written it out, I see it maybe doesn't buy us much. Feel free to take it or leave it :laughing:

Reply:
I think it makes sense, I'll make the change